hibernate-core/src/test/java/org/hibernate/orm/test/mappingexception/MappingExceptionTest.java

```diff
@@ -111,15 +111,6 @@ public void testNotFound() throws MappingException, MalformedURLException {
 			assertEquals( "nothere", e.getOrigin().getName() );
 		}
 
-		try {
-			cfg.addResource( "nothere", getClass().getClassLoader() );
-			fail();
-		}
-		catch (org.hibernate.boot.MappingNotFoundException e) {
-			assertEquals( SourceType.RESOURCE, e.getOrigin().getType() );
-			assertEquals( "nothere", e.getOrigin().getName() );
-		}
-
```

Comment:
Why was this test removed?

Reply:
The Configuration.addResource() method that accepts a ClassLoader was removed since it was deprecated, so testing this call didn't seem to make sense any more

Reply:
Right, but the previous implementation just ignored the passed class loader, so you could just omit the class loader in these two tests as well.

Reply:
If you mean that there should be a call to Configuration.addResource (String resourceName), that's still there in the test ... unless i'm misunderstanding your intention ... ?

Reply:
You are right, I didn't see the call in the lines before this. It's fine!